src/kernel-install/kernel-install

```diff
@@ -182,32 +169,22 @@ case $COMMAND in
             fi
         fi
 
-        for f in "${PLUGINS[@]}"; do
-            if [[ -x $f ]]; then
-                [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && \
-                    echo "+$f add $KERNEL_VERSION $ENTRY_DIR_ABS $KERNEL_IMAGE ${INITRD_OPTIONS[@]}"
-                "$f" add "$KERNEL_VERSION" "$ENTRY_DIR_ABS" "$KERNEL_IMAGE" "${INITRD_OPTIONS[@]}"
-                x=$?
-                if [[ $x == $SKIP_REMAINING ]]; then
-                    break
-                fi
-                ((ret+=$x))
-            fi
+        for f in $PLUGINS; do
+            [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "+$f add $KERNEL_VERSION $ENTRY_DIR_ABS $KERNEL_IMAGE $*"
+            "$f" add "$KERNEL_VERSION" "$ENTRY_DIR_ABS" "$KERNEL_IMAGE" "$@"
```

Comment:
I think losing `INITRD_OPTIONS` is an unnecessary loss of leggibility.

Reply:
Whereas I can read this just fine! Amazing what fluency in a language does!

Reply:
Fluency is a wonderful thing, but code is also about communicating intent to other people and the next person looking at this might not well-versed in POSIX shell or be in a hurry. There's a reason to give variables sensible names. In this case one has to follow the whole script to follow mutations of `$@` throughout. A simple copy earlier on would make it easy to see at a glance what this is.

Reply:
If you aren't well-versed in shell then you shouldn't be writing a program in shell without testing it throughout. Also, there's literal 300 lines of XML describing what this line does in intricate detail. Either way, this is a made-up problem.

Reply:
I'd prefer to keep INITRD_OPTIONS too. It does make the code more readable. And no, we can't assume that every person who reads this code or even changes it is fluent in shell. Some complexity and required implicit knowledge cannot be avoided, but we should avoid that wherever possible.